gamestonk_terminal/stocks/stocks_controller.py

```diff
@@ -254,23 +254,27 @@ def call_news(self, other_args: List[str]):
         parser.add_argument(
             "-s",
             "--sources",
-            dest="n_sources",
             default=[],
             nargs="+",
-            help="Show news only from the sources specified (e.g bbc.com yahoo.com)",
+            help="Show news only from the sources specified (e.g bbc yahoo.com)",
```

Comment:
you're missing a comma here between bbc and yahoo

Reply:
Doesn't "nargs=+" eliminate the need for commas?

Reply:
You're right. I saw the `",".join()` and got confused, but we just process it like that after. So all good